fiber-colly-gorm/db/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:14.1-alpine
+
+ADD create_db.sql /docker-entrypoint-initdb.d
```

Comment:
Instead of this file, just mount `create_db.sql` in the docker compose. That way you only need the base image

Reply:
Changed them all as you requested. If there are any other changes, just give me a shout.